content/en/docs/setup/install/operator/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Operator Install [Experimental]
+description: Install and configure Istio using the Istio Operator.
+weight: 25
+keywords: [operator,kubernetes,helm]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Follow this guide to install and configure an Istio mesh using an alternate
+installation method: the Istio {{<gloss operator>}}Operator{{</gloss>}}
+installation.
+
+The Istio Operator installation is a shorter process with the option of
+installing Istio using a one-line command. It has user input
+validation to help prevent installation errors and customization options to
+override any aspect of the configuration.
+
+It has the following additional characteristics:
+
+- Descriptive error messages to spot configuration errors easier.
+
+- Installer capable of waiting for Istio pods and services to be in a
+  ready state.
+
+- Fewer dependencies (doesn’t require Helm).
+
+The Operator install is accessed via [`istioctl`](/docs/reference/commands/istioctl/)
+commands and a new Operator API.
+
+## Prerequisites
+
+Before you begin, you must meet the following prerequisites.
+
+1. [Download the Istio release](/docs/setup/#downloading-the-release).
+1. Perform any necessary [platform-specific setup](/docs/setup/install/platform/).
+1. Check the [Requirements for Pods and
+   Services](/docs/setup/additional-setup/requirements/).
+
+## Install Istio using One Line
+
+The simplest option is to install Istio using a one-line command. At the command
+prompt, run the following command:
+
+{{< text bash >}}
+$ istioctl manifest apply
+{{< /text >}}
+
+This command installs a profile named `default` on the cluster defined by your
+Kubernetes configuration. The `default` profile is smaller and more suitable
+for establishing a production environment, unlike the larger profile named
+`demo` that is meant to evaluate a broad set of Istio features.
+
+You can override the `default` profile using the `--kubeconfig` flag.
```

Comment:
this seems a little odd from a usability perspective.  I'd expect `--kubeconfig` to specify the kubeconfig context under use, rather than specifyigin the profile under use.

@ostromart can you weigh in?

Reply:
yeah @sdake is right, --kubeconfig is for kube config :-)
To select a profile you'd use --set profile=<name> or -f <customization file with profile: set>

Reply:
Removed --kubeconfig flag mention

Reply:
addressed